doc/threadsafe_function_ex.md

```diff
@@ -186,110 +188,99 @@ Returns one of:
 
 ```cpp
 #include <chrono>
-#include <thread>
 #include <napi.h>
+#include <thread>
 
 using namespace Napi;
 
 using Context = Reference<Value>;
 using DataType = int;
-void CallJs( Napi::Env env, Function callback, Context* context, DataType* data );
+void CallJs(Napi::Env env, Function callback, Context *context, DataType *data);
 using TSFN = ThreadSafeFunctionEx<Context, DataType, CallJs>;
 using FinalizerDataType = void;
 
 std::thread nativeThread;
 TSFN tsfn;
 
-Value Start( const CallbackInfo& info )
-{
+Value Start(const CallbackInfo &info) {
   Napi::Env env = info.Env();
 
-  if ( info.Length() < 2 )
-  {
-    throw TypeError::New( env, "Expected two arguments" );
-  }
-  else if ( !info[0].IsFunction() )
-  {
-    throw TypeError::New( env, "Expected first arg to be function" );
-  }
-  else if ( !info[1].IsNumber() )
-  {
-    throw TypeError::New( env, "Expected second arg to be number" );
+  if (info.Length() < 2) {
+    throw TypeError::New(env, "Expected two arguments");
+  } else if (!info[0].IsFunction()) {
+    throw TypeError::New(env, "Expected first arg to be function");
+  } else if (!info[1].IsNumber()) {
+    throw TypeError::New(env, "Expected second arg to be number");
   }
 
   int count = info[1].As<Number>().Int32Value();
 
   // Create a new context set to the the receiver (ie, `this`) of the function
   // call
-  Context* context = new Reference<Value>( Persistent( info.This() ) );
```

Comment:
nit: This is actually fine and not in need of changing. It conforms to the Node.js coding style.